docs/topics/api/addons.rst

```diff
@@ -457,10 +457,16 @@ on AMO.
         it's up to the clients to re-order results as needed before displaying
         the add-ons to the end-users.
 
+        In addition, the results can be cached for up to 24 hours, based on the
+        full URL used in the request.
+
     :query string app: Mandatory. Filter by :ref:`add-on application <addon-detail-application>` availability.
+    :query string appversion: Filter by application version compatibility. Pass the full version as a string, e.g. ``46.0``. Only valid when both the ``app`` and ``type`` parameters are also present, and only makes sense for Language Packs, since Dictionaries are always compatible with every application version.
```

Comment:
This is a little ambiguous if you can specifiy `appversion` without specifying `type` - i.e. get filtered langpacks+unfiltered dicts in one response.

Reply:
You can't. Specifying `appversion` forces you to specify `type`, we can't do appversion filtering and return both types at the same time currently.